lib/que/active_support/job_middleware.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Que
+  module ActiveSupport
+    module JobMiddleware
```

Comment:
Some usage documentation would be in order, I reckon.

Reply:
Have added to readme do you think its better here?

Reply:
Ah, nah, I just wanted to start a thread so it'd be able to be marked resolved :laughing:

Cheers for adding that; looks good